keywords/LiteServiOS.py

```diff
@@ -74,16 +72,19 @@ def install_device(self):
         Warning: Only works with a single device at the moment
         """
 
-        if self.storage_engine != "SQLite":
-            raise LiteServError("https://github.com/couchbaselabs/liteserv-ios/issues/1")
+        if self.storage_engine == "SQLCipher":
+            package_name = "LiteServ-iOS-SQLCipher-Device.app"
+            app_dir = "LiteServ-iOS-SQLCipher"
+        else:
+            package_name = "LiteServ-iOS-Device.app"
+            app_dir = "LiteServ-iOS"
 
-        package_name = "LiteServ-iOS.app"
-        app_path = "{}/{}/{}".format(BINARY_DIR, "LiteServ-iOS", package_name)
-        log_info("Installing: {}".format(app_path))
+        self.app_path = "{}/{}/{}".format(BINARY_DIR, app_dir, package_name)
```

Comment:
In this file, I modified app_path to self.app_path and for device to self.device as I need to access on other method i.e open app, so I need as global variable